### PR TITLE
Update a few things

### PR DESCRIPTION
**Pull request by krischer**
* Migrates to a `pyproject.toml` based package description.
* Removes most version restrictions - the `pyqtgraph` related ones might need a deeper look.
* Fixed a few tests.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+[project]
+authors = [
+    { name = "SCCER-SoE" },
+    { name = "SED" },
+    { name = "ETHZ" },
+    { name = "Joseph Doetsch", email = "joseph.doetsch@erdw.ethz.ch" },
+    { name = "Linus Villiger", email = "linus.villiger@sed.ethz.ch" },
+    { name = "Thomas Haag" },
+    { name = "Sem Demir" },
+    { name = "DUGSeis Authors" },
```
Comment (jadoetsch) [inline pyproject.toml:14]: This list of authors reflects a very early version of the library. Should this be changed to or extended with the authors of the current paper?

File: pyproject.toml [hunk 2]
```diff
@@ -1,3 +1,64 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    { name = "SCCER-SoE" },
+    { name = "SED" },
+    { name = "ETHZ" },
+    { name = "Joseph Doetsch", email = "joseph.doetsch@erdw.ethz.ch" },
```
Comment (jadoetsch) [inline pyproject.toml:10]: My email can be removed (as it is also not valid anymore).

File: pyproject.toml (hunk 1 above)
Comment (jadoetsch) [reply pyproject.toml:14]: @mrosskopf can you adjust the package author list?

**Review by jadoetsch — COMMENTED** [review]
Looks goot to me.

@mrosskopf can you check the library author list? It is quite outdated and e.g. doesn't include Lion and you.

**Comment by mrosskopf** [comment]
I integrated the changes manually.

**Pull request closed without merging**